src/utils/ast.js

```diff
@@ -7,21 +7,22 @@
 import { without, range } from "lodash";
 
 import type { Location, Source, ColumnPosition } from "../types";
+import type { Symbols } from "../reducers/ast";
 
-import type {
-  AstPosition,
-  AstLocation,
-  PausePoint,
-  SymbolDeclarations
-} from "../workers/parser";
+import type { AstPosition, AstLocation, PausePoint } from "../workers/parser";
 
 export function findBestMatchExpression(
-  symbols: SymbolDeclarations,
+  symbols: Symbols,
   tokenPos: ColumnPosition
 ) {
-  const { memberExpressions, identifiers, literals } = symbols;
-  const { line, column } = tokenPos;
+  if (
+    !(symbols.memberExpressions && symbols.identifiers && symbols.identifiers)
+  ) {
+    return null;
```

Comment:
> !(symbols.memberExpressions && symbols.identifiers && symbols.identifiers)

could we just do `!symbols`

Reply:
well no because as per type def for Symbols it can be either SymbolDeclaration | {| loading: true |}. So we have to account for the other case. Really it seems like we are only dealing with SymbolDeclarations here. What do you think?